source/fab/source_tree.py

```diff
@@ -27,42 +27,58 @@ def visit(self, candidate: Path) -> List[Path]:
         raise NotImplementedError("Abstract method must be implemented")
 
 
-class ExtensionVisitor(TreeVisitor):
+class PathMap(object):
```

Comment:
I think ultimately the PathMap class should represent a single mapping and the list should be of those objects. For the moment this is fine.

Reply:
That does make sense - I will keep it in mind when updating #80 after this (as that will require a little extra work to this area anyway)